getting-started/installation-windows-env.rst

```diff
@@ -13,12 +13,12 @@ Required settings for Path
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
 The way you are building your binaries determines which paths have to be added to the Path variable.
 
-* Static builds
+* Static builds (For more Details see `Product Static Build <https://docs.epics-controls.org/en/latest/appdevguide/EPICSBuildFacility.html#product-static-builds>`_.)
 
-  1. Add the EPICS Base binary directory for your target to be able to call the EPICS command line tools without specifying their fully qualified path.
+  1. Add the EPICS Base binary directory ("\\base-x.x.x\\bin") for your target to be able to call the EPICS command line tools without specifying their fully qualified path.
```

Comment:
The binary directory should be `...\base-x.x.x\bin\<target-arch>`, shouldn't it?

Reply:
Yes. I will update.